isis/src/base/apps/spiceinit/spiceinit.cpp

```diff
@@ -662,12 +657,17 @@ namespace Isis {
         continue;
       }
     }
+    
+    if (QString(ui.GetString("SHAPE")).toLower() == "user") {
```

Comment:
The toLower should not be necessary. When the UI/AML parameter is associated with a list of possible values (i.e. radio buttons) it always returns the uppercase version of the of the list element it uniquely matches. So change the litteral to uppercase and remove the toLower. Note, free form strings are not modified, but then why would we be doing a string comparison on a free form string?